src/components/TimeTable/timeTable.js

```diff
@@ -1,7 +1,26 @@
 import React from "react";
 
 export default function TimeTable() {
+
     return (
-        <h1>TimeTable</h1>
+        <div className="table">
+            <button onClick={loadTable()}>Kliknij</button>
+            <div className="content"></div>
+        </div>
+
+
     )
+    function loadTable()  {
+        var xhttp = new XMLHttpRequest();
+        xhttp.onreadystatechange = function (){
+            if(this.readyState === 4 && this.status === 200){
+                document.getElementsByClassName("content").innerHTML = this.responseText;
+            }
+        }
+        xhttp.open("POST","timeTable.php",true);
```

Comment:
Do zmiany "timetable.php" na ścieżkę bezwzględną. Można napisać komponent, który będzie automatycznie doklejał resztę URLa

Reply:
Tylko tu sie pojawia problem że to chyba nie może być plik na dysku ale nie wiem :v